### DISCUSSION TITLE
Can't find extension after installing it

### DISCUSSION THREAD

**Discussion by bflanagan2** [zanllp/infinite-image-browsing#466] [General]
Sry if this is a newbie question but... I'm a bit lost.  Where do I find this extension after I've installed it?  I installed it via the Extensions tab in my A111 instance (which is running on EC2), clicked "Apply and Restart UI", the app refreshed but I'm not seeing anything different. The extension is showing as active (the checkbox next to it is checked).  Is there a new link or button to click to access this extension? Is there a custom route that I should use in the URL?

**Comment by Manniala** [comment]
![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/32381860/951c14ca-badb-429d-afe2-2e07b9f0102e)
It should be at the top

Also try to look under Settings >> User Interface, see if you have put it in the Hiden UI Tabs, it should than be listed here
![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/32381860/3ccf0cbe-d05d-4a7f-bc51-298a79a6aae2)
